contrib/mesos/pkg/proc/proc.go

```diff
@@ -122,11 +121,13 @@ func (self *procImpl) begin() runtime.Signal {
 	var entered runtime.Latch
 
 	// execute actions on the backlog chan
+	self.wg.Add(1)
```

Comment:
this package has been completely re-implemented, see #10457 (lgtm'd, awaiting merge)

can we avoid making changes here for now?